app/views/unattended/provisioning_templates/cloud_init/cloud_init_default.erb

```diff
@@ -0,0 +1,39 @@
+<%#
```

Comment:
This needs to go through community templates too. I guess it helps with the review, but we should sync it from there to make sure, we have the latest version and next sync won't causd regressions.

Reply:
Isn't this part of https://github.com/theforeman/community-templates/pull/514?

Reply:
Yes, but what i mean is that if that PR changes, we may forgot to update it in this PR too. So I think it would be better to drop it from here and after this is merged, do the sync using `script/sync_templates.sh`. Or do it as part of this PR but just before merge.

Reply:
I think we should just merge the community-templates PR in sync with this PR. I like having the templates here so we automatically run snapshot & seed tests with the templates.

Reply:
Let's simply agree that templates in this PR are just copies of the c-t repository, that's the master. Just avoid reviewing the templates here in this PR.

Reply:
The one who merges this PR must make sure that the c-r repo PR is correct (the same content or it works).